SpinCore_pp/config_parser_fn.py

```diff
@@ -0,0 +1,90 @@
+from pyspecdata import *
+import configparser
+
+class configuration(object):
+    # this registers the type, the pretty case we want, the section, and
+    # whether or not we can assume a default value
+    registered_params = {
+        "amplitude":(float, "acq_params", None),
+        "deadtime_us":(float, "acq_params", None),
+        "tau_us":(float, "acq_params", None),
+        "deblank_us":(float, "acq_params", None),
+        "SW_kHz":(float, "acq_params", None),
+        "acq_time_ms":(float, "acq_params", None),
+        "carrierFreq_MHz":(float, "acq_params", None),
+        "startconstant":(float, "acq_params", None),
+        "stopconstant":(float, "acq_params", None),
+        "p90_us":(float, "acq_params", None),
+        "gamma_eff_MHz_G":(float, "acq_params", 0.00425),
+        "mw_freqs":(float,"acq_params",9.821e9), # JF asks -- what is this in contrast to the uw_...?
+        "concentration":(float, "sample_params", None),
+        "krho_cold":(float, "sample_params", None),
+        "krho_hot":(float, "sample_params", None),
+        "T1water_hot":(float, "sample_params", None),
+        "T1water_cold":(float, "sample_params", None),
+        "repetition_us":(float, "sample_params", None),
+        "guessed_MHz_to_GHz":(float,"sample_params", 1.51671), # the ppt value we use to determine our rf carrier frequency based on our microwave
```

Comment:
why is this a sample parameter? this isn't used in data acquisition nor processing...